### DISCUSSION TITLE
Frontend

### DISCUSSION THREAD

**Discussion by lonely0712** [thedaviddias/Front-End-Checklist#518] [General]
Hi guys. I want tour help on something... What is the roadmap to be a frontend developer. Right know I know the basics html, css, javascript and react. What do you recommend me to learn right know?

**Comment by 3boodon** [comment]
Since you've already learned HTML, CSS, JS and react, I would recommend learning more about the following: 
- Restful / GraphQl APIs
- Typescript 
- Unit Tests, E2E tests, integration tests ... etc
- Web Accessiblty 
- SEO
- PWA 
- Web Sockets 
- SSR, SSG 

This is what comes to my mind at the moment. Anyways, the most important part is to build projects to apply what you learn.

**Reply by lonely0712** [reply]
Thanks
